### DISCUSSION TITLE
I have an idea I can do web development in HTML without any errors and I mentioned cognitive problems. Is HTML a good choice for me?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#141850] [Programming Help]
### Body

I have an idea I can do web development in HTML without any errors and I mentioned cognitive problems. Is HTML a good choice for me?

### Guidelines

- [ ] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Zuba43** [comment]
Html isn't bad, except that you won't bc that selfishness to write all the code manually when we've code generators like WordPress, Dreamweaver and may more. Just get basics of HTML and the rest use a code generator

**Comment by muhammadashraf23** [comment]
@WorldsDumbestCoder Yes, HTML is a great choice, especially if you're looking for something straightforward and easy to understand. Since you mentioned having cognitive challenges, HTML's simplicity can be beneficial. It’s a markup language rather than a programming language, which means it’s mainly about structuring content rather than dealing with complex logic.

**Reply by ghost** [reply]
```javascript
console.log("What do I do now if I already learned HTML, CSS and JavaScript?");
```

**Comment by Himani0002** [comment]
Is that possible Web development without Html ? please tall me ?

**Reply by muhammadashraf23** [reply]
> Is that possible Web development without Html ? please tall me ?

Technically, web development without HTML isn't possible because HTML (HyperText Markup Language) is the fundamental building block of the web. Every web page is ultimately rendered in a browser as HTML, even if the original source code is written in another format or language.

**Reply by MuxN4** [reply]
HTML is like the skeleton of a website, it gives structure to everything. Without it, the web wouldn’t really exist in its current form. Even if you're using other tools or frameworks, HTML is still the foundation, think of it like how the frame of a house holds everything together, even if you don't see it. So, yes, web development without HTML isn't really possible.

**Comment by YuheshPandian** [comment]
Try exploring everything about HTML then maybe you would have a clue that you need to use other languages for certain functions.
